src/pages/personalisation/LifeSkillsSubcategories.js

```diff
@@ -12,9 +12,9 @@ export default class LifeSkillsSubcategories extends BaseMultiQuestion {
         answers: {
             /* eslint-disable max-len */
             "Daily living": remove("life skills education").append("daily living skills"),
-            "Personal": remove("life skills education").append("personal development"),
-            "School": remove("life skills education").append("(school education) -private")
-                .append("-catholic -independent -care -recreation -pathways"),
+            "Personal": remove("life skills education").append("personal development")
+                .append("-vcal - grants"),
```

Comment:
Also since both these words have `-`, should `append` be changed to `remove` and the `-` be taken away?